src/sdcp_printer/__init__.py

```diff
@@ -298,6 +318,30 @@ def _send_request(
             )
         )
 
+    async def send_request_async(
```

Comment:
I like exposing this as a "public" function, but would like to see it tied into the private backer, or making the private public.